src/pages/index.js

```diff
@@ -36,9 +38,41 @@ const initialCards = [
     link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/spots/7-photo-by-griffin-wooldridge-from-pexels.jpg",
   },
 ];
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "26a66bc9-8fc9-40b5-980e-7373cd063117",
+    "Content-Type": "application/json",
+  },
+});
+
+// Destructure the second item in the callback of the .then()
+api
+  .getAppInfo()
+  .then(([cards, userInfo]) => {
+    profileName.textContent = userInfo.name;
+    profileDescription.textContent = userInfo.about;
+    const profileAvatar = document.querySelector(".profile__avatar");
```

Comment:
`    const profileAvatar = document.querySelector(".profile__avatar");
`

You've already found `avatarModalBtn` in line 75. Please, use it here

Reply:
My remark hasn't been corrected